scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -1034,7 +1042,7 @@ class MicrobeEditor{
     private int symmetry = 0;
 
     private bool microbeHasBeenInEditor = false;
-
+    private bool nucleusIsPresent = false;
```

Comment:
As far as I can tell this doesn't work if the nucleus is added before entering the editor.

Reply:
Well yes, but can nucleus be adess before?

Reply:
You can add the nucleus, exit the editor (as you have to as you are out of MP) and then the next time everything will be locked again.

Reply:
Why everything should be locked? you mean i can't add more organelles?

Reply:
https://github.com/Revolutionary-Games/Thrive/issues/679

Reply:
Ok so to summarize, if i haven't nucleus i can add only non membrane organelles, when i have nucleus i can add them.

Reply:
Yes